api/libs/planning-area-repository/src/custom-planning-area.repository.ts

```diff
@@ -51,6 +51,14 @@ INSERT INTO "planning_areas"("the_geom")
   }
 
   async assignProject(id: string, projectId: string): Promise<void> {
+    await this.planningAreas.update(
```

Comment:
in practice we could even delete this planning area straight away rather than orphaning it and letting it be GC'ed later on (but we'd need to do this in a transaction to make sure the delete can be rolled back if the successive update fails